pom.xml

```diff
@@ -37,6 +37,7 @@
         <module>integrationtest/wildfly</module>
         <module>integrationtest/performance</module>
         <module>integrationtest/osgi/karaf-features</module>
+        <module>integrationtest/osgi/karaf-it</module>
```

Comment:
why did you do this? There is a profile 'integration-karaf' which gets activates for JDK 7 and 8. The test does not work on JDK 9, but Sanne still wants to build against it, so he added this additional profile. Admittedly one can easily get lost with all the profiles defined in this pom. I would not mind a clean-up (in particular removing the db profiles, but that's a different story).

If you are ok with it,  I remove this line and merge. Other than that all looks good.